packages/cmf-cqrs/src/hooks.ts

```diff
@@ -0,0 +1,65 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { WEBSOCKET_READY_STATE } from './constants';
+import { getWebSocketFromRegistry } from './registry';
+
+export interface WebsocketMessageData {
+	applicationId: string;
+	applicationService: string;
+	body: any;
+	clientId: string;
+	messageType: string;
+	timestamp: number;
```

Comment:
I believe we don't need this. It seems specific and not used anywhere, so keep the generic behind to let the user specify the types seems enough

Reply:
I would like to keep the `body` / `messageType` part if it's ok for you.
Otherwise, for the denyList, i would go for a custom filter function that could be passed to the hook (would work also :))